packages/grafana-ui/src/components/GraphNG/nullInsertThreshold.test.ts

```diff
@@ -115,23 +115,6 @@ describe('nullInsertThreshold Transformer', () => {
     expect(result.fields[2].values.toArray()).toStrictEqual(['a', null, 'b', null, 'c']);
   });
 
-  test('should insert nulls at midpoints between adjacent > interval: 2', () => {
```

Comment:
this wasnt actually testing the thing it said it was and was just a duplicate test